regions/io/core.py

```diff
@@ -137,7 +137,7 @@ def to_crtf(self,  coordsys='fk5', fmt='.6f', radunit='deg'):
             'line': '{0}line[[{1:FMT}deg, {2:FMT}deg], [{3:FMT}deg, {4:FMT}deg]]'
                         }
 
-        output = '#CRTF\n'
+        output = '#CRTFv0\n'
```

Comment:
This document: https://casaguides.nrao.edu/index.php/CASA_Region_Format says `#CRTF` is the expected prefix, not `#CRTFv0`.  Since we don't know what version we're outputting, we should not add the `v0`

Reply:
Using the mask with heading #CRTF produces the following message:
`2020-05-14 13:51:50     WARN    task_tclean::RegionTextParser::_determineVersion        RegionTextParser::_determineVersion: File ../Reduction_scripts/ARTIP/J1833_mask.crtf does not contain a CASA Region Text File spec version. The current spec version, 0 will be assumed. WE STRONGLY SUGGEST YOU INCLUDE THE SPEC VERSION IN THIS FILE TO AVOID POSSIBLE FUTURE BACKWARD INCOMPATIBILTY ISSUES. Simply ensure the first line of the file is '#CRTFv0'`

It's not an error to be fair, but maybe it's worth considering.

Reply:
OK, yes, let's add the `v0` now.  This is something that has been updated in the spec: the more recent versions of the doc include the `v0` suffix.

I would put money on there never being a `v>0` though =)